ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewBackgroundDrawable.java

```diff
@@ -343,30 +343,59 @@ private void drawRectangularBackgroundWithBorders(Canvas canvas) {
       int width = getBounds().width();
       int height = getBounds().height();
 
+      // If the path drawn previously is of the same color,
+      // there would be a slight white space between borders
+      // with anti-alias set to true.
+      // Therefore we need to disable anti-alias, and
+      // after drawing is done, we will re-enable it.
+
+      mPaint.setAntiAlias(false);
+
+      Path borderPath = new Path();
```

Comment:
Can you just allocate this once up in the constructor and reset it each time it is used?